### DISCUSSION TITLE
Margin and sidebar fold option

### DISCUSSION THREAD

**Discussion by AxelDgn** [quarto-dev/quarto-cli#9533] [Q&A]
### Description

Hello,

I'm currently working on a quarto book and I would like to know if there is an option to fold the sidebar and margin to the side to have more space for the body and be able to unfold these elements with a button.

Thank you in advance for your help !

**Comment by mcanouil** [comment]
There is no direct/native option for this, but this could probably be done with CSS (maybe a bit of JavaScript), see <https://getbootstrap.com/docs/5.3/components/collapse/>.
